### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by gogainda**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,23 +10,17 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: ${{matrix.image}}
     strategy:
       fail-fast: false
       matrix:
-        image:
-          - ruby:2.5.0
```
Comment (pointlessone) [inline .github/workflows/ci.yml:18]: I appreciate your enthusiasm about TruffleRuby but I'd like to keep our current set of rubies as well.

Comment (gogainda) [reply .github/workflows/ci.yml:18]: I will revert it back, just want to skip all other ruvies for now

Comment (pointlessone) [reply .github/workflows/ci.yml:18]: OK. Let me know when this PR is ready.

**Comment by gogainda** [comment]
@pointlessone  tests are passing, but rubocop fails due to a [bug](https://github.com/oracle/truffleruby/issues/1913), so without upgrading rubocop to the 0.90 truffleruby cannot be enabled right now

**Comment by gogainda** [comment]
I tried to upgrade the rubocop version to 0.90 and it helped, but it introduced some failing checks which are part of newer version of rubocop.

**Comment by eregon** [comment]
@gogainda Could you just skip RuboCop on TruffleRuby? It would be redundant anyway and just make the build slower.

**Comment by pointlessone** [comment]
I tried adding truffleruby in the latest CI update and encountered some issues.

Specifically, build fails with truffleruby and the failure is some sort of error in Rubocop.

At the moment I don't see truffleruby feasible. Especially since there's no one on the team using it. We'll gladly accept help from someone who's willing to spend their time regularly fixing compatibility issues.

**Comment by eregon** [comment]
Makes sense, I'll make a draft PR and try to get it working.
It would be valuable to test latest Prawn + TruffleRuby since it's a non-trivial codebase and actually all tests pass fine.

**Comment by eregon** [comment]
https://github.com/prawnpdf/prawn/pull/1189

**Pull request closed without merging**